### DISCUSSION TITLE
Social Sharing - Cookbooks Make Better Use of "Favorite Recipes"

### DISCUSSION THREAD

**Discussion by noja11** [mealie-recipes/mealie#2719] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

My users and myself have amassed a literal ton of recipes over the last year (around 1000). Each user has also narrowed down their favourites by clicking the heart on a recipe. My users and I would like an easy way to share our favourites with each other. Currently the best method is to create a share link and send it, which is difficult and doesn't give me a list of what my sister likes to make. The other option is to have each user create add a custom tag on each of their favourite recipes and then create a Cookbook that filters on that custom tag.

### Please provide a concise description of the feature that would resolve your issue.

If possible, add in a few more social aspects to the app so that users can see other users and certain things like favourites, when they last made a recipe, etc. 

Adjust the Cookbook creation options to allow filtering on a user's Favorite Recipes.

### Please consider and list out some caveats or tradeoffs made in your design decision

Adding in more social stuff is probably difficult, but could be fun.  Food is inherently social and recipes are interesting conversation topics. Truthfully though, the easiest way to offer this specific functionality would be to allow Cookbooks to use a user's favourites. 

Thanks! 

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by tdharris** [comment]
I too would love to see this! Cookbooks are a "cross section of Categories, Tags, and Tools" and it would be helpful if there was potential to also include **Favorites** in that cross section. It would be nice to be able to create a Cookbook of "My Favorites" that included all the recipes that I have simply added to my Favorites. I could then easily share that cookbook. I find it's quicker and easier to favorite a recipe than it is to Edit and scroll to the bottom to manage the categories, tags, and tools.

**Comment by michael-genson** [comment]
You are able to view other users' favorites using the direct link for favorites: `https://your-mealie-url.com/user/userId/favorites` (where `userId` is the user's id)

I agree this should be easier to access, though

**Comment by mblom** [comment]
Let me start by expressing my gratitude for your effort in developing this tool! I just recently installed on my server and I really like it a lot!

For my family it would be great too, to be able to search/filter recipes that are marked a favorite.

**Comment by mschut** [comment]
Yes! This feature would be awesome. We are using Mealie within our family, and it would be really nice to be able to see each others favorites as well. The ratings are personal eaither, it would be nice if we each could rate the recipes, but we would also be able to see all the scores and/or the overall rating. 

Would be really nice if you could search also within each others favorites/select them. For example: create a cookbook where User favorite is true (or from a certain user, or which is favorite of at least two users etc…).

**Comment by Professor-64** [comment]
Fully agreed. I would also like to have the option to manually add recipes to a cookbook. That way, I can easily share recipes without having to add a tag everywhere. I see cookbooks as “photo albums,” but for recipes.
